fms/tests/test_fms.py

```diff
@@ -0,0 +1,147 @@
+import os
+
+import pytest
+from happi import Client
+from happi.backends.json_db import JSONBackend
+from happi.errors import SearchError
+from happi.item import OphydItem
+
+from fms.__main__ import add_fms_sensor, add_src_controller, delete_sensor
+
+from ..happi.containers import FMSSRCItem
+from .test_utils import assert_valid, sensor_in_list
+
+test_file = "test.db"
+
+
+@pytest.fixture
+def clean_files(monkeypatch):
+    db = JSONBackend(path=test_file, initialize=True)
+    client = Client(path=test_file)
+    sensor = [("temp1", "3"), ("temp2", "4")]
+
+    inputs = iter(["N", "TST:PV", "R64", "L88", "L77", "TST", "TST", "class"])
+    monkeypatch.setattr("builtins.input", lambda _: next(inputs))
```

Comment:
Note/FYI: 

This is a neat built-in way of testing these input prompt workflows.  In `happi` we use `click`, which comes with a rather helpful method for this purpose:  [(docs)](https://click.palletsprojects.com/en/stable/testing/#input-streams)

See: https://github.com/pcdshub/happi/blob/26ea6c182d06d95970c590ed3c7dc2883e937c85/happi/tests/test_cli.py#L546-L550